BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -253,84 +394,6 @@ public static string GetSecdrvVersion(string firstMatchedString, IEnumerable<str
             }
         }
 
-        // TODO: Continue collecting SHA-1 hashes instead of sizes
-        private string GetVersionFromSHA1Hash(string sha1Hash)
-        {
-            switch (sha1Hash.ToLowerInvariant())
-            {
-                // dplayerx.dll
-                case "f7a57f83bdc29040e20fd37cd0c6d7e6b2984180":
```

Comment:
Where did most of these hashes go? I can't see the `dplayerx.dll` and `secdrv.sys` ones anywhere

Reply:
Hashes were never a good indication for these files, if any are found to be reliable they'll be added from scratch in the next stage.

Reply:
Please keep the ones that have been gathered, then. Even if they're not used, they're useful for now.